{
    "body": "I am releasing v2.1.8.0 for SZ, also modifying package.py by adding maintainer and parallel=False.\r\nI ran 'spack test' but failed. However, it seems that failure has nothing to do with my modification. I just modified sz/package.py. The failures are 'test_url_parse', 'test_url_local_file_path', 'test_url_join_local_paths'. ",
    "user": "disheng222",
    "url": "https://api.github.com/repos/spack/spack/issues/13974",
    "updated_at": "2019-12-04 06:02:54",
    "created_at": "2019-12-04 02:54:17",
    "closed_at": "2019-12-04 04:35:47",
    "state": "closed",
    "title": "Release a new version v2.1.8.0 for SZ",
    "number": 13974,
    "milestone": null,
    "labels": [
        "new-version"
    ],
    "id": 532414510,
    "html_url": "https://github.com/spack/spack/pull/13974",
    "assignees": [],
    "comments": 3
}